proc/proc_windows.go

```diff
@@ -117,7 +117,7 @@ func Launch(cmd []string) (*Process, error) {
 	// after launching under DEBUGONLYTHISPROCESS.
 	var tid, exitCode int
 	dbp.execPtraceFunc(func() {
-		tid, exitCode, err = dbp.waitForDebugEvent()
+		tid, exitCode, err = dbp.waitForDebugEvent(true)
```

Comment:
Unrelated to your review, but "Only the thread that created the process being debugged can call WaitForDebugEvent." (see https://msdn.microsoft.com/en-us/library/windows/desktop/ms681423(v=vs.85).aspx). The thread that calls WaitForDebugEvent here (the one running handlePtraceFuncs) is different from the thread that called CreateProcess just few lines above.